src/database/remove.ts

```diff
@@ -0,0 +1,93 @@
+"use strict";
+
+import * as pathLib from "path";
+import { Queue } from "../queue";
+import FirebaseError = require("../error");
```

Comment:
```suggestion
import * as FirebaseError from "../error";
```

applies to all the `require` statements here